scripts/release/go_update/run_GO_Update.pl

```diff
@@ -0,0 +1,92 @@
+#!/usr/local/bin/perl  -w
+use strict;
+
+use lib '/usr/local/gkb/modules';
+use GKB::Config;
+
+use autodie qw/:all/;
+use Cwd;
+use Getopt::Long;
+use Data::Dumper;
+use File::Basename;
+use List::MoreUtils qw/uniq/;
+
+use Log::Log4perl qw/get_logger/;
+
+Log::Log4perl->init(\$LOG_CONF);
+my $logger = get_logger(__PACKAGE__);
+
+my $data_release_pipeline_application_version = "1.0.0";
+my $data_release_pipeline_application = "go-update";
+my $data_release_pipeline_tag = "go-update-1.0.0";
+my $data_release_pipeline_repository = "https://github.com/reactome/data-release-pipeline";
+my $resource_dir = "data-release-pipeline/".$data_release_pipeline_application."/src/main/resources/";
+my $start_directory = cwd(); # abs_path($0);
+
+my $release_version = @args[0];
+
+print "start_directory: ".$start_directory."\n";
+
+if (-e "data-release-pipeline") {
+  $logger->info("data-release-pipeline exists, pulling");
+  # Start on develop. Pull will fail if we're not already on a branch
+  chdir "data-release-pipeline";
+  system("git checkout develop");
+  if (!$data_release_pipeline_tag) {
+    system("git pull");
+  # if there is a tag defined, we should check that out.
+  } else {
+    #chdir "data-release-pipeline";
+    # Still need to do a pull, since the tag might be been created after the last pull
+    system("git pull");
+    system("git checkout ".$data_release_pipeline_tag);
+    #chdir "..";
+  }
+  chdir $start_directory;
+  system("cp ".$data_release_pipeline_application.".properties $resource_dir");
```

Comment:
Just FYI, Perl will interpolate variables in double quotes, so they can go in the String without concatenating.  The .properties can be put on the end of the $data_release_pipeline_application variable inside the quotes too since the dot isn't an accepted character for a variable name, Perl will know how to parse the variable name correctly.

Reply:
Yeah, it's pretty conventional when it comes to permitted variable name characters :).

https://www.webassign.net/manual/instructor_guide/t_i_setting_perl_variables.htm

Reply:
Perl doesn't like being predictable